File: 프로그래머스/0/120839. 가위 바위 보/가위 바위 보.java
class Solution {
    public String solution(String rsp) {
        String answer = "";
        rsp = rsp.replaceAll("2","s").replaceAll("0","r").replaceAll("5","p");
        answer = rsp.replaceAll("s","0").replaceAll("r","5").replaceAll("p","2");
        return answer;
    }
}

File: 프로그래머스/0/120839. 가위 바위 보/가위 바위 보.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

char* solution(const char* rsp) {
    size_t len = strlen(rsp);
    char* answer = (char*)malloc((len + 1) * sizeof(char));
    
    for(int i = 0; i < len; i++){
        if(rsp[i] == '2') {
            answer[i] = '0';
            continue;
        }
        if(rsp[i] == '0') {
            answer[i] = '5';
            continue;
        }
        if(rsp[i] == '5') {
            answer[i] = '2';
            continue;
        }
    }
    
    answer[len] = '\0';
    
    return answer;
}